FilePersistence/src/version_control/merge/ChangeGraph.cpp

```diff
@@ -448,28 +446,113 @@ void ChangeGraph::removeLabelConflictsBetweenChildren(Model::NodeIdType parentId
 	}
 }
 
-void ChangeGraph::updateIncomingLabels(Model::NodeIdType /*parentId*/, IdToLabelMap /*labelMap*/)
+void ChangeGraph::updateLabelsOfChangesTo(Model::NodeIdType parentId, IdToLabelMap labelMap, GenericTree* tree)
 {
-	Q_ASSERT(false);
-}
+	auto parentNode = tree->find(parentId);
+	Q_ASSERT(parentNode != NULL);
```

Comment:
Hmm, actually, it seems like we only need the tree here, so that we can perform this check. I would rather remove the tree argument and not do this check, and let this function work purely on the merge changes.